pkg/storage/storagewrappers/boundedconcurrency.go

```diff
@@ -0,0 +1,82 @@
+package storagewrappers
+
+import (
+	"context"
+	"time"
+
+	"github.com/openfga/openfga/pkg/storage"
+	"github.com/prometheus/client_golang/prometheus"
+	"github.com/prometheus/client_golang/prometheus/promauto"
+	openfgapb "go.buf.build/openfga/go/openfga/api/openfga/v1"
+	"go.opentelemetry.io/otel/attribute"
+	"go.opentelemetry.io/otel/trace"
+)
+
+const timeWaitingSpanAttribute = "time_waiting"
+
+var _ storage.RelationshipTupleReader = (*boundedConcurrencyTupleReader)(nil)
+
+var (
+	timeWaitingHistogram = promauto.NewHistogram(prometheus.HistogramOpts{
```

Comment:
Can we name the variable after the metric name please?

`boundedReadDelayMsHistogram` or `boundedReadDelayHistogram`